modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID arrays using the `mdadm` utility, and integrate the RAID arrays with {lvms}.
```

Comment:
```suggestion
Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID arrays using the `mdadm` utility, and integrate the RAID array device with {lvms}.
```

Reply:
I see that RAID array is used but I'm not sure I understand why if RAID is an array it isn't in backticks.

Reply:
I referred to the RAID documentation in RHEL, where backticks are not used for RAID. Just followed the same here  as well. Not sure if backticks should be used for RAID.